Readme.md

```diff
@@ -131,14 +132,21 @@ requests for consideration or create an issue with a code change proposal.
 
 ### Local testing
 
-There are currently no automatic tests available. The most simple way for at
-least syntax correctness checks is to just call `./bot-ng.py --help` to show
-the help text if the source can be correctly parsed. The next recommended way
-for testing is to call `bot-ng.py` with the `--dry` command line parameter in
-different modes. This might need additional data, e.g. "metadata" from
-https://gitlab.suse.de/qa-maintenance/metadata/ . For example with cloning
-this metadata as well as specifying a fake token value that is enough for
-testing:
+There are currently only limited automatic tests available. Call
+
+```
+make test
```

Comment:
Standard in normal python projets is simply call `pytest`

Reply:
which does not work because you opted to remove the one code line in https://github.com/openSUSE/qem-bot/pull/21#discussion_r857422438

Reply:
So, `make test` doesn't help, because it just calls `py.test`, which doesn't work here for me.

I don't understand what you two are doing here :)

Reply:
Thank you for fixing the call with https://github.com/openSUSE/qem-bot/pull/25 . I guess it was a misunderstanding between mimi1vx and me in #21 and #22